src/shared/services/workbasket/workbasket-input-filter.service.ts

```diff
@@ -32,9 +32,8 @@ export class WorkbasketInputFilterService {
     this.currentCaseType = caseTypeId;
     return this.httpService
       .get(url, {headers, observe: 'body'})
-      .map(response => {
-        let jsonResponse = response;
-        let workbasketInputs = jsonResponse.workbasketInputs;
+      .map(body => {
+        let workbasketInputs = body.workbasketInputs;
```

Comment:
My favourite thing... can this be a `const` instead of a `let`? 😄

Reply:
Done. Pushed.